### PR TITLE
refactor(helpers): rename & replace deleteMessageByID() with deleteMessage()

### PR DESCRIPTION
**Pull request by ayntee**

### CODE REVIEW

File: src/helpers/messages/delete_message.ts [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
-/** Delete a message */
+/** Delete a message with the channel id and message id only. */
 export async function deleteMessage(
-  message: Message,
+  messageID: string,
   reason?: string,
   delayMilliseconds = 0,
 ) {
+  const message = await cacheHandlers.get("messages", messageID);
+  if (!message) throw new Error(Errors.MESSAGE_NOT_FOUND);
```
Comment (Skillz4Killz) [inline src/helpers/messages/delete_message.ts:16]: A vast majority of the time u wont have the message cached. We should not throw this.

```ts
if (message) {
 // checks here
 if (message.author.id !== botID){}
}

```

File: src/types/errors.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ export enum Errors {
   // Message Delete Errors
   DELETE_MESSAGES_MIN = "DELETE_MESSAGES_MIN",
   PRUNE_MIN_DAYS = "PRUNE_MIN_DAYS",
+  MESSAGE_NOT_FOUND = "MESSAGE_NOT_FOUND",
```
Comment (Skillz4Killz) [inline src/types/errors.ts:22]: Not really needed imo

File: src/helpers/messages/delete_message.ts (hunk 1 above)
Comment (ayntee) [reply src/helpers/messages/delete_message.ts:16]: We are also using `message.channelID` property... :thinking:

File: src/types/errors.ts (hunk 2 above)
Comment (ayntee) [inline src/types/errors.ts:22]:
```suggestion

```

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**